client/src/components/client/sider/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import 'antd/dist/antd.css';
+import './index.css';
+import { Layout, Menu, ConfigProvider } from 'antd';
+import { WalletFilled, BankFilled, SettingFilled } from '@ant-design/icons';
+
+const { Content, Sider } = Layout;
+const SiderMenu = () => (
+  <ConfigProvider direction="rtl">
+    <div className="sider-wrapper">
+      <Layout dir="rtl">
+        <Sider dir="rtl" className="sider-rtl">
+          <Menu
+            theme="light"
+            className="side-menu"
+            mode="inline"
+            defaultSelectedKeys={['4']}
+          >
+            <Menu.Item key="1" icon={<WalletFilled />}>
+              المحفظة
+            </Menu.Item>
+            <Menu.Item key="2" icon={<BankFilled />}>
+              حساب البنك
+            </Menu.Item>
+            <Menu.Item key="3" icon={<SettingFilled />}>
+              إعدادت
+            </Menu.Item>
```

Comment:
```suggestion
            <Menu.Item key="1" icon={<WalletFilled />}>
             محفظتي
            </Menu.Item>
            <Menu.Item key="2" icon={<BankFilled />}>
             تفاصيل البنوك
            </Menu.Item>
            <Menu.Item key="3" icon={<SettingFilled />}>
              الإعدادات
            </Menu.Item>
```
Also , make the font size larger and choose better font family for writing, besides that you have to leave a place for image and select box as its in our design.